staging/src/k8s.io/api/apps/v1beta2/types.go

```diff
@@ -294,7 +293,7 @@ type DeploymentSpec struct {
 
 	// Label selector for pods. Existing ReplicaSets whose pods are
 	// selected by this will be the ones affected by this deployment.
-	// +optional
+	// It must match the pod template's labels.
 	Selector *metav1.LabelSelector `json:"selector,omitempty" protobuf:"bytes,2,opt,name=selector"`
```

Comment:
Would you remove `omitempty` tags too? Ref
https://github.com/kubernetes/community/blob/master/contributors/devel/api-conventions.md#optional-vs-required

Reply:
@janetkuo Addressed， PTAL